user/enterprise/operations-manual.md

```diff
@@ -106,6 +106,24 @@ The above mentioned error can be caused by a configuration mismatch in [the GitH
 
 Your Travis CI Enterprise license has a hostname field which contains the hostname of your installation. When the license's hostname does not match the actual hostname, the container does not start. If this is the case or you suspect that this might be likely, please [get in touch with us](mailto:enterprise@travis-ci.com?subject=License%20Hostname%20Change) and we'll help you to get back on track.
 
+
+## travis-worker on 16.04 does not start
```

Comment:
Here the issue is that travis-worker does not start. Then we need to "filter" down to Ubuntu 16.04, so 

> travis-worker does not start on Ubuntu 16.04